packages/api/lambdas/bulk-operation.js

```diff
@@ -118,26 +128,67 @@ function applyWorkflowToGranules({
 async function bulkGranuleDelete(payload) {
   const granuleIds = await getGranuleIdsForPayload(payload);
   const granuleModel = new GranuleModel();
+  const granulePgModel = new GranulePgModel();
   const forceRemoveFromCmr = payload.forceRemoveFromCmr === true;
+  const knex = await getKnexClient({ env: process.env });
   const deletedGranules = [];
   await pMap(
     granuleIds,
     async (granuleId) => {
+      const collectionPgModel = new CollectionPgModel();
+
+      let dynamoGranule;
+      let pgGranule;
+
+      // Try to get the Dynamo record. If it cannot be found, just log
+      // the error and skip it.
       try {
-        const granule = await granuleModel.getRecord({ granuleId });
-        if (granule.published && forceRemoveFromCmr) {
-          await granuleModel.unpublishAndDeleteGranule(granule);
```

Comment:
This should be removed from the granule model if it's no longer being used.

Reply:
Yeah, I have a lot of squeamishness about removing Dynamo methods. My fear is that they're exposed and used somewhere that I'm not seeing. But that shouldn't be possible if I can't find references.

Getting over it 😄 

https://github.com/nasa/cumulus/pull/2108/commits/bb49238ba4da92a741dcc8bb32d3b23872ceaf13

Reply:
You're actually correct this should be *deprecated* using @cumulus/common/util/deprecate as it's publically exported. 

There's a bit of absurdity to that stance given our efforts to remove Dynamo, but it might be worth a brief discussion.  

Regardless of our choice here we should include a CL entry as well.

Reply:
Yeah, I think I agree with your initial instinct to just remove this one. If deleting it involved changing a dozen references over to this new logic, that might not be something we'd want to tackle without a lot of consideration. In this case though the only reference to `unpublishAndDeleteGranule` is here and we've removed it. I think it's safe to delete it from the model entirely.

Updated the changelog to call it out:
https://github.com/nasa/cumulus/pull/2108/commits/3ad39cc1a62ed481e419e57387fc4f7622b0ac59

Reply:
The issue being we have a precedent of *always* deprecating anything that's a 'public'/exported interface, and api/models/* is.